api.rst

```diff
@@ -1452,6 +1452,68 @@ You can use a tool like `Swagger UI <https://swagger.io/tools/swagger-ui/>`_ to
 
   The OpenAPI information can go out of date as the schema changes under a running server. To learn how to refresh the cache see :ref:`schema_reloading`.
 
+.. _options_requests:
+
+OPTIONS
+=======
+
+You can verify which HTTP methods are allowed on endpoints for tables and views using an OPTIONS request. These methods are allowed depending on what operations *can* be done on the table or view, not on the database permissions assigned to them.
+
+CORS
+----
+
+PostgREST sets highly permissive cross origin resource sharing, that is why it accepts Ajax requests from any domain.
+
+Tables
+------
+
+Suppose you make this API request to an existing table named ``people``.
+
+.. code-block:: http
+
+  OPTIONS /people HTTP/1.1
+
+Then, the response you get is:
+
+.. code-block:: http
+
+  HTTP/1.1 200 OK
+  Allow: OPTIONS,GET,HEAD,POST,PUT,PATCH,DELETE
+
+
+Views
+-----
+
+The methods that can be used on views are determined by the presence of INSTEAD OF TRIGGERS and primary key columns:
+
+.. table::
+   :widths: auto
+
+   +--------------------+-------------------------------------------------------------------------------------------------+
+   | Method allowed     | View's requirements                                                                             |
+   +====================+=================================================================================================+
+   | OPTIONS, GET, HEAD | None (Always allowed)                                                                           |
+   +--------------------+-------------------------------------------------------------------------------------------------+
+   | POST               | INSTEAD OF INSERT TRIGGER                                                                       |
+   +--------------------+-------------------------------------------------------------------------------------------------+
+   | PUT                | INSTEAD OF INSERT TRIGGER, INSTEAD OF UPDATE TRIGGER, primary key columns                       |
```

Comment:
```suggestion
   | PUT                | INSTEAD OF INSERT TRIGGER, INSTEAD OF UPDATE TRIGGER, also requires the presence of a  primary key                       |
```